bindings.cpp

```diff
@@ -3358,13 +3207,6 @@ bool DeRestPluginPrivate::checkSensorBindingsForAttributeReporting(Sensor *senso
         }
         else if (*i == VENDOR_CLUSTER_ID)
         {
-            if (sensor->modelId().startsWith(QLatin1String("RWL02")) || // Hue dimmer switch
-                sensor->modelId().startsWith(QLatin1String("ROM00")) || // Hue smart button
-                sensor->modelId().startsWith(QLatin1String("RDM00")) || // Hue wall switch module
-                sensor->modelId().startsWith(QLatin1String("Z3-1BRL"))) // Lutron Aurora Friends-of-Hue dimmer switch
-            {
-                val = sensor->getZclValue(*i, 0x0000); // button event
-            }
             if (sensor->modelId() == QLatin1String("de_spect")) // dresden elektronik spectral sensor
```

Comment:
The `de_spect` will be removed in a followup PR. It was a internal prototype spectral sensor for a student measuring rgb parts of the light spectrum. If necessary it can be added later as DDF.